src/sentry/web/forms/accounts.py

```diff
@@ -223,6 +223,19 @@ class RegistrationForm(PasswordlessRegistrationForm):
         required=True, widget=forms.PasswordInput(attrs={"placeholder": "something super secret"})
     )
 
+    recaptcha = forms.CharField(
+        required=False,
+        widget=forms.HiddenInput(
+            attrs={
+                "id": "recaptcha",
+                "data-should-add-recaptcha": (
+                    "true" if settings.SHOULD_ADD_GOOGLE_RECAPTCHA_V3 else "false"
```

Comment:
I think we could probably come up with a better name for this. Maybe `USE_GOOGLE_RECAPTCH_V3` ?